claudetool/bash.go

```
@@ -169,8 +171,106 @@ func executeBash(ctx context.Context, req bashInput) (string, error) {
+		return "", fmt.Errorf("failed to start pty: %w", err)
+	}
+	defer ptmx.Close()
+
+	proc := cmd.Process
+	done := make(chan struct{})
+	go func() {
+		select {
+		case <-ctx.Done():
+			if ctx.Err() == context.DeadlineExceeded && proc != nil {
+				// Kill the entire process group.
+				syscall.Kill(-proc.Pid, syscall.SIGKILL)
```

golangci-lint (1.64.8)

Error return value of `syscall.Kill` is not checked

(errcheck)